controller/src/main/java/io/pravega/controller/store/stream/ZKStoreHelper.java

```diff
@@ -59,7 +59,7 @@ public ZKStoreHelper(final CuratorFramework cf, Executor executor) {
             } catch (KeeperException.NodeExistsException e) {
                 throw StoreException.create(StoreException.Type.NODE_EXISTS, path);
             } catch (Exception e) {
-                throw StoreException.create(StoreException.Type.UNKNOWN, path);
+                throw new StoreException.UnknownException(path, e);
```

Comment:
Basically it defines the class of errors that do not fall into any known error types. I am not sure whether `UNSPECIFIED` suits the bill.